components/identifier-generator/back/tests/.php_cd.php

```diff
@@ -39,7 +39,12 @@
             'Akeneo\Pim\Enrichment\Component\Product\Validator\Constraints\Product\UniqueProductEntity',
             'Akeneo\Pim\Structure\Component\AttributeTypes',
             'Akeneo\Pim\Structure\Component\Query\PublicApi',
+            'Akeneo\Pim\Structure\Family\ServiceAPI\Query\FindFamiliesWithLabels',
+            'Akeneo\Pim\Structure\Family\ServiceAPI\Query\FamilyQuery',
+            'Akeneo\Pim\Structure\Family\ServiceAPI\Query\FamilyQueryPagination',
+            'Akeneo\Pim\Structure\Family\ServiceAPI\Query\FamilyQuerySearch',
```

Comment:
```suggestion
            'Akeneo\Pim\Structure\Family\ServiceAPI',
```

It's ServiceAPI, you're allowed to be coupled with everything in this folder.